pkg/awsclient/ec2_helper.go

```diff
@@ -111,9 +139,6 @@ func (c *AwsClient) removeIngressRulesFromSecurityGroup(securityGroup *ec2.Secur
 		return nil
 	}
 	i := &ec2.RevokeSecurityGroupIngressInput{
-		FromPort:      aws.Int64(6443),
-		ToPort:        aws.Int64(6443),
-		IpProtocol:    aws.String("tcp"),
```

Comment:
This is required because one may have either these attributes _or_ the `IpPermissions` attribute.